Preference/Mondu/Mondu/Model/Request/Transactions.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mondu\MonduPaymentHyva\Preference\Mondu\Mondu\Model\Request;
+
+use Hyva\Checkout\Model\ConfigData\HyvaThemes\Checkout as HyvaCheckoutConfig;
+use Magento\Checkout\Model\Session as CheckoutSession;
+use Magento\Framework\HTTP\Client\Curl;
+use Magento\Framework\Locale\Resolver;
+use Magento\Framework\UrlInterface;
+use Magento\Quote\Model\Cart\CartTotalRepository;
+use Magento\Quote\Model\Quote;
+use Magento\Quote\Model\Quote\Address;
+use Mondu\Mondu\Helpers\BuyerParams\BuyerParamsInterface;
+use Mondu\Mondu\Helpers\Logger\Logger as MonduFileLogger;
+use Mondu\Mondu\Helpers\OrderHelper;
+use Mondu\Mondu\Model\Request\Transactions as OriginalTransactions;
+use Mondu\Mondu\Model\Ui\ConfigProvider;
+
+class Transactions extends OriginalTransactions
+{
+    private string $fallbackEmail;
+
+    public function __construct(
+        Curl $curl,
+        CartTotalRepository $cartTotalRepository,
+        CheckoutSession $checkoutSession,
+        ConfigProvider $configProvider,
+        private readonly OrderHelper $orderHelper,
+        private readonly UrlInterface $urlBuilder,
+        private readonly BuyerParamsInterface $buyerParams,
+        private readonly Resolver $store,
+        private readonly MonduFileLogger $monduFileLogger,
+        private readonly HyvaCheckoutConfig $checkoutConfig,
+    ) {
+        parent::__construct(
+            $curl,
+            $cartTotalRepository,
+            $checkoutSession,
+            $configProvider,
+            $orderHelper,
+            $urlBuilder,
+            $buyerParams,
+            $store,
+            $monduFileLogger
+        );
+    }
+
+    public function request($_params = []): array
+    {
+        try {
+            if ($_params['email']) {
+                $this->fallbackEmail = $_params['email'];
+            }
+            $params = $this->getRequestParams();
+
+            if (in_array( $_params['payment_method'], array('direct_debit', 'installment', 'installment_by_invoice'))) {
```

Comment:
can we create constants for payment methods  so that they are reusable and we don't have to hardcode the strings everywhere?